Docsp 8721 mongoose vs driver pg (#717)
mongodb/docs-node: source/index.txt

version for each MongoDB Server version, see the
:doc:`Compatibility </compatibility>` section.

Comparing the Node.js Driver and Mongoose
-----------------------------------------

> Learn the difference between the Node.js Driver and the third-party
> library Mongoose in the :ref:`<node-driver-vs-mongoose>` section.

Reviewer: ```suggestion
Learn about how the {+driver-long+} compares with Mongoose, a third-party ODM, in the :ref:`node-driver-vs-mongoose` section.
```
